sp-utilities/creator-node/healthChecks.js

```diff
@@ -191,17 +183,85 @@ async function healthCheckFileUpload () {
       headers: {
         ...sampleTrack.getHeaders()
       },
-      url: `${CREATOR_NODE_ENDPOINT}/health_check/fileupload`,
+      url: `${CREATOR_NODE_ENDPOINT}/transcode_and_segment`,
       method: 'post',
-      params: reqParam,
+      params: {
+        spID: SP_ID,
+        timestamp,
+        signature
+      },
       responseType: 'json',
       data: sampleTrack,
       maxContentLength: Infinity,
       maxBodyLength: Infinity
     }
-    resp = await axios(requestConfig)
+    let transcodeResp = await axios(requestConfig)
+    const uuid = transcodeResp.data.data.uuid
+
+    const MAX_TRACK_TRANSCODE_TIMEOUT = 3600000 // 1 hour
+    const POLL_STATUS_INTERVAL = 3000 // 3s
+    let TRANSCODE_STATE = { successful: false, data: null }
+
+    // this is pulled directly from pollProcessingStatus in libs/services/creatorNode/CreatorNode.ts
+    const start = Date.now()
+    while (Date.now() - start < MAX_TRACK_TRANSCODE_TIMEOUT) {
+      try {
+        const processingStatusResp = await axios({
+          url: `${CREATOR_NODE_ENDPOINT}/async_processing_status`,
+          params: {
+            uuid
+          },
+          method: 'get'
+        })
+        const { status } = processingStatusResp.data.data
+
+        if (status && status === 'DONE') {
+          TRANSCODE_STATE.successful = true
+          TRANSCODE_STATE.data = processingStatusResp.data.data
+          break
+        }
+        if (status && status === 'FAILED') {
+          console.error(`Track content async upload failed: uuid=${uuid}, error=`, processingStatusResp)
+        }
+      } catch (e) {
+        // Catch errors here and swallow them. Errors don't signify that the track
+        // upload has failed, just that we were unable to establish a connection to the node.
+        // This allows polling to retry
+        console.error(`Failed to poll for processing status, ${e}`)
+      }
+
+      await wait(POLL_STATUS_INTERVAL)
+    }
+
+    if (!TRANSCODE_STATE.successful) {
+      throw new Error(
+        `Track content async upload took over ${MAX_TRACK_TRANSCODE_TIMEOUT}ms. uuid=${uuid}`
+      )
+    }
+
+    console.log("Successfully got the transcode response for uuid: ", uuid)
+
+    // clear the track data on the node
+    let clearRequestConfig = {
+      url: `${CREATOR_NODE_ENDPOINT}/clear_transcode_and_segment_artifacts`,
+      method: 'post',
+      params: {
+        spID: SP_ID,
+        timestamp,
+        signature
+      },
+      data: {
+        fileDir: TRANSCODE_STATE.data.resp.fileDir
+      },
+      responseType: 'json'
+    }
+    const clearArtifactsResp = await axios(clearRequestConfig)
```

Comment:
wouldn't this throw if it doesn't 200? so we wouldnt see the thrown err on line 260

Reply:
maybe wrap in try/catch if we want a particular err msg

Reply:
we actually don't need the explicit 200 check, axios will throw for non 2XX response codes so i'll just delete the if statement